crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -2126,6 +2118,8 @@ struct RSATestVector kRSATestVectors[] = {
     { 2048, &EVP_sha256, false, AWSLC_APPROVED, AWSLC_APPROVED },
     { 2048, &EVP_sha384, false, AWSLC_APPROVED, AWSLC_APPROVED },
     { 2048, &EVP_sha512, false, AWSLC_APPROVED, AWSLC_APPROVED },
+    { 2048, &EVP_sha512_224, false, AWSLC_APPROVED, AWSLC_APPROVED },
```

Comment:
The SHA3 NIDs should be added here as well pending https://github.com/aws/aws-lc/pull/1805 being merged.

Reply:
Done